### ISSUE TITLE
Change defaults to build on more architectures

### ISSUE THREAD

**Issue by trbocode** [rocm-arch/python-pytorch-rocm#50]
This is not as much as an issue as a light pkgbuild problem - by default this package is built only for 4 architectures. I think it would be best to at least follow AMDs rules and add gfx1030.

**Comment by acxz** [comment]
I agree, I think it also makes sense to build it for as many architectures that work.
Can you point to the source on what architecture pytorch wheels are built for? We can start with that.

**Comment by trbocode** [comment]
gfx90a and gfx1030 are surely being built from (1030 works on wheel and 90a is current CDNA), but I don't know more than just files which suggest current ones and those two.

**Comment by acxz** [comment]
> just files which suggest current ones and those two

That's fine! If you could show that here that would be great. Maybe the output of a grep command on the list of files in the unpackaged wheel?

I just want to have a link/trail that we can point to as reference.

**Comment by trbocode** [comment]
➜  torch fd gfx
lib/library/Kernels.so-000-gfx1030.hsaco
lib/library/Kernels.so-000-gfx803.hsaco
lib/library/Kernels.so-000-gfx900.hsaco
lib/library/Kernels.so-000-gfx906-xnack-.hsaco
lib/library/Kernels.so-000-gfx908-xnack-.hsaco
lib/library/Kernels.so-000-gfx90a-xnack+.hsaco
lib/library/Kernels.so-000-gfx90a-xnack-.hsaco
lib/library/TensileLibrary_gfx1030.co
lib/library/TensileLibrary_gfx803.co
lib/library/TensileLibrary_gfx900.co
lib/library/TensileLibrary_gfx906.co
lib/library/TensileLibrary_gfx908.co
lib/library/TensileLibrary_gfx90a.co

From an unpackaged wheel.

**Comment by N00byKing** [comment]
[As of July 7](https://github.com/pytorch/pytorch/issues/73534#issuecomment-1178040772), its `gfx900, gfx906, gfx908, gfx90a, gfx1030`.
Would personally also really look forward to gfx1030 support!
